server/api/commands.go

```diff
@@ -276,18 +274,19 @@ func Help() string {
 	io.ReplyNL(w, io.Magenta+"        -m, --make"+io.Grey+" runs make")
 	io.ReplyNL(w, io.Magenta+"        -v, --verbose"+io.Grey+" shows the output")
 	io.ReplyNL(w, io.Grey+"    when using docker, the only applicable option is -v, all the others are implicitly used")
-	io.ReplyNL(w, io.Magenta+"test <duration> <events> <spans> <frames> <concurrency> [<apmserver-flags> <OPTIONS>...]")
+	io.ReplyNL(w, io.Magenta+"test <duration> <events> <spans> <frames> <agents> [<apmserver-flags> <OPTIONS>...]")
 	io.ReplyNL(w, io.Grey+"    starts the apm-server and performs a workload test against it")
 	io.ReplyNL(w, io.Magenta+"        <duration>"+io.Grey+" duration of the load test (eg \"1m\")")
-	io.ReplyNL(w, io.Magenta+"        <events>"+io.Grey+" events per request, either transactions or errors")
-	io.ReplyNL(w, io.Magenta+"        <spans>"+io.Grey+" spans per transaction: if 0 events are errors, otherwise they are transactions")
-	io.ReplyNL(w, io.Magenta+"        <frames>"+io.Grey+" frames per document, either spans or errors")
-	io.ReplyNL(w, io.Magenta+"        <concurrency>"+io.Grey+" number of simultaneous queries to send")
+	io.ReplyNL(w, io.Magenta+"        <transactions>"+io.Grey+" transactions per request")
+	io.ReplyNL(w, io.Magenta+"        <spans>"+io.Grey+" spans per transaction")
+	io.ReplyNL(w, io.Magenta+"        <frames>"+io.Grey+" frames per document (for spans and errors)")
+	io.ReplyNL(w, io.Magenta+"        <agents>"+io.Grey+" number of simultaneous agents sending queries")
```

Comment:
the actual option is `num_agents`, so you either change `num_agents` to `agents` or change it in the help from `agents` to `num_agents` (i like `agents` more)